core/src/main/scala/kafka/log/remote/RemoteLogManager.scala

```diff
@@ -592,7 +587,7 @@ class RemoteLogManager(fetchLog: TopicPartition => Option[Log],
           }
 
           //b. cleanup/delete expired remote segments
-          handleExpiredRemoteLogSegments()
+          // handleExpiredRemoteLogSegments()
```

Comment:
Disabling the remote log deletion until the feature is ready. This breaks the existing tests.